ci/do_ci.sh

```diff
@@ -50,16 +50,27 @@ if [[ "$1" == "bazel.release" ]]; then
     echo 'Ignoring build for git tag event'
     exit 0
   fi
-
+  
   setup_gcc_toolchain
   echo "bazel release build with tests..."
   bazel_release_binary_build
-  echo "Testing..."
-  # We have various test binaries in the test directory such as tools, benchmarks, etc. We
-  # run a build pass to make sure they compile.
-  bazel --batch build ${BAZEL_BUILD_OPTIONS} -c opt //include/... //source/... //test/...
-  # Now run all of the tests which should already be compiled.
-  bazel --batch test ${BAZEL_TEST_OPTIONS} -c opt //test/...
+  
+  if [[ $# > 2 ]]; then
+  	echo "Multiple tests are not supported, only $2 will run"
```

Comment:
nit: inconsistent indent. Also, couldn't we run more than 2 if we do some shifting of args and then wildcard arguments?

Reply:
Right, it is quite simple, did not think of it. Do you think this option should be added to other targets beside release?